tokio/CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+# 0.3.0 (October 15, 2020)
+
+This represents a 1.0 beta release. APIs are polished and future-proofed. APIs
+not included for 1.0 stabilization have been removed.
+
+Biggest changes are:
+
+- I/O driver internal rewrite. The windows implementation includes significant
+  changes.
+- Runtime API is polished, especially with how it interacts with feature flag
+  combinations.
+- Feature flags are simplified
+  - `rt-core` and `rt-util` are combined to `rt`
+  - `rt-threaded` is renamed to `rt-multi-thread` to match builder API
+  - `tcp`, `udp`, `uds`, `dns` are combied to `net`.
+  - `parking_lot` is included with `full`
+
+### Changes
+- meta: Minimum supported Rust version is now 1.47.
```

Comment:
Uh, we currently run `minrust` test at 1.45:

https://github.com/tokio-rs/tokio/blob/30b40ef518d1767736c8ff47c794481e88bfd3e9/.github/workflows/ci.yml#L13

Reply:
Given the compile time issues with 1.46 and 1.47 (https://github.com/rust-lang/rust/issues/75992), it's probably better to keep support for 1.45 for now.

Reply:
I guess we can and bump further for 1.0. I mostly want access to `Default` impl for larger arrays :)